datalake-api/src/main/java/com/sequenceiq/sdx/api/endpoint/SdxEndpoint.java

```diff
@@ -255,4 +256,10 @@ RangerCloudIdentitySyncStatus getRangerCloudIdentitySyncStatus(@PathParam("envCr
     @Produces(MediaType.APPLICATION_JSON)
     @ApiOperation(value = ROTATE_CERTIFICATES, nickname = "rotateAutoTlsCertificatesByCrn")
     FlowIdentifier rotateAutoTlsCertificatesByCrn(@PathParam("crn") String crn, @Valid CertificatesRotationV4Request rotateCertificateRequest);
+
+    @POST
+    @Path("/customimage/{name}")
```

Comment:
Can you please follow the pattern? we use this pattern: `{name}/operation_name` or `{crn}/operation_name`. Our URL parser won't recognise the pattern.

Reply:
@topolyai5 Thanks for the comment. Will @Path("{name}/customimage") work? If so, I will make update tomorrow. 

Is this the only change I need to make? If not, can we have 30 minutes zoom session to discuss and get all changes we need at once, then I can work on it on Monday. Thanks!

Reply:
I mean for this path: `{name}/custom_image`

Reply:
maybe it can work with crn as well?

Reply:
@topolyai5 I don't think it work with crn. This is create function. the caller can only provide the DL name. The crn is not assigned yet. To get the details, we can use the crn, and actually reuse the existing function to do so. 

The added new function is only create() with customized image info.

Reply:
yes, you are right